molecule/cookiecutter/scenario/driver/docker/{{cookiecutter.molecule_directory}}/{{cookiecutter.scenario_name}}/INSTALL.rst

```diff
@@ -7,10 +7,9 @@ Requirements
 
 * General molecule dependencies (see https://molecule.readthedocs.io/en/latest/installation.html)
 * Docker Engine
-* docker-py
-* docker
+* Docker python library https://docker-py.readthedocs.io/en/stable/
 
 Install
 =======
 
-    $ sudo pip install docker-py
+    $ sudo pip install docker
```

Comment:
We've added extras, so it's even like this:
```suggestion
     $ pip install molecule[docker]
```

and please wipe `sudo` from every place you edit (ref #1579)

Reply:
Great !

Reply:
done.